el/src/main/java/com/sun/ts/tests/el/api/jakarta_el/arrayelresolver/ELClientIT.java

```diff
@@ -33,26 +32,48 @@
 import jakarta.el.ArrayELResolver;
 import jakarta.el.ELContext;
 
-public class ELClient extends ServiceEETest {
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+
+import java.lang.System.Logger;
+
+
+public class ELClientIT extends ServiceEETest {
+
+  private static final Logger logger = System.getLogger(ELClientIT.class.getName());
 
   private Properties testProps;
 
   private static final String[] names = { "doug", "nick", "roger", "ryan",
       "ed" };
 
-  public static void main(String[] args) {
-    ELClient theTests = new ELClient();
-    Status s = theTests.run(args, System.out, System.err);
-    s.exit();
+  // public static void main(String[] args) {
```

Comment:
good to remove jtest related unused code.

Reply:
Done